examples/demo.ipynb

```diff
@@ -161,7 +161,8 @@
     "    2. RGB/RGBA (example: `color=(0.1,0.1,0.1)`, `color=(0.1,0.1,0.1,0.1)`)\n",
     "    3. Hex (example: `color=\"#eeefff\"`)\n",
     "    4. Random (example: `color=\"random\"`)\n",
-    "    5. Complement (example: `color=\"complement\", bgcolor=\"blue\"`)"
+    "    5. Complement (example: `color=\"complement\", bgcolor=\"blue\"`)\n",
+    "    6. Transparent (example: `bgcolor=\"transparent\"`)"
    ]
```

Comment:
Please update here with this sentence:

⚠️ **Transparent** mode is only available for background